apps/molgenis-components/src/components/forms/Tooltip.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <span class="mg-tooltip-container">
+    <slot />
+    <div v-if="value" class="mg-tooltip">
+      <div class="mg-tooltiptext">
+        {{ value }}
+      </div>
+    </div>
+  </span>
+</template>
```

Comment:
bootstrap has a tooltip component https://getbootstrap.com/docs/4.1/components/tooltips/

Reply:
yeah, I tried to use that one but was hard to operate without their javascript magic. Therefore chose another component

Reply:
suggest to use the Bootstap component and wire it using vue